testFramework/com/twitter/intellij/pants/testFramework/PantsCodeInsightFixtureTestCase.java

```diff
@@ -79,9 +79,7 @@ protected void setUp() throws Exception {
       assertTrue("Failed to enable Python plugin!", PluginManagerCore.enablePlugin(pyPluginId));
     }
 
-    final String testUserHome = VfsUtil.pathToUrl(FileUtil.toSystemIndependentName(PantsTestUtils.BASE_TEST_DATA_PATH + "/userHome"));
-    final Optional<VirtualFile> folderWithPex =
-      PantsUtil.findFolderWithPex(Optional.ofNullable(VirtualFileManager.getInstance().findFileByUrl(testUserHome)));
+    final Optional<VirtualFile> folderWithPex = PantsUtil.findFolderWithPex();
```

Comment:
In tests we've been using a pants.pex file commited to the repository. We can either change the test to use the pex file in user's home, or copy the file from user's home to `testData/userHome/.pants.d/bin/pants.pex/`. IMO the second approach would be better, because it would be more explicit what is accessed from $HOME/.cache

Reply:
On the other hand the first approach would closer to real-world behavior

 @lukaszwawrzyk @kamildoleglo @wisechengyi